ios/TempleWallet.xcodeproj/project.pbxproj

```diff
@@ -661,7 +661,7 @@
 				COPY_PHASE_STRIP = YES;
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
why do you have these changes?

Reply:
because of M1 processor, it makes simulator working, but here i tried to undo changes